README.rst

```diff
@@ -1213,6 +1213,17 @@ Choose from the list of available rules:
   - ``sortAlgorithm`` (``'alpha'``, ``'none'``): how multiple occurrences of same type
     statements should be sorted; defaults to ``'none'``
 
+* **ordered_implements**
+
+  Orders the interfaces in an ``implements`` clause.
```

Comment:
would sorting multiple `extends` of an interface very similar to the logic in this fixer?

Reply:
Would need additional checks to see if it applies and where to fix. (Aka `isCandidate` and the current first lines of `applyFix`.)

The rest of the logic should be the same indeed.

And now you gonna ask me to add that as well? 😜

Reply:
it can wait, after this one maybe some abstraction can be done to create an additional fixer so we would have both ;)

Reply:
Code-wise my preference would be separate. However as a user, I think single fixer would be better. Would you do one without doing the other? But giving the option is nice as well.

I can see it as both a separate fixer or as part of this one, so I'll leave that up to you guys to decide.

Reply:
I would go with a single fixer, but that means implementing it in this PR as it would probably require to rename the fixer.

Reply:
single fixer makes sense to me :)
that would mean:
- rename the fixer
- candidate detection would become 'implements' or 'extends && interface' is found
- update the check if candidate token in the fix method
- I wouldn't update the config, I don't see why someone would sort `implements` and `extends` differently, but sure up for discussion ;)

Reply:
> single fixer makes sense to me :)
> that would mean:
 
- [X] rename the fixer
- [X] candidate detection would become 'implements' or 'extends && interface' is found
- [X] update the check if candidate token in the fix method
- [X] I wouldn't update the config
- [X] update code samples
- [X] update tests
- [X] update readme
- [ ] push commit